tests/fast-integration/telemetry-test/mockserver-resources.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mockserver
+  name: mockserver
+  namespace: mockserver
+spec:
+  ports:
+  - name: serviceport
+    port: 1080
+    targetPort: serviceport
+  selector:
+    app: mockserver
+  type: ClusterIP
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: mockserver-config
+  name: mockserver-config
+  namespace: mockserver
+data:
+  initializerJson.json: |
+    [
+      {
+        "httpRequest": {
+          "path": "/"
+        },
+        "httpResponse": {
+          "statusCode": 201,
+          "body": "some response"
+        }
+      }
+    ]
+  mockserver.properties: |
+    mockserver.maxSocketTimeout=120000
+    mockserver.dynamicallyCreateCertificateAuthorityCertificate=true
+    mockserver.directoryToSaveDynamicSSLCertificate=.
+    mockserver.sslCertificateDomainName=localhost
+    mockserver.sslSubjectAlternativeNameDomains=www.example.com,www.another.com
+    mockserver.sslSubjectAlternativeNameIps=127.0.0.1
+    mockserver.enableCORSForAPI=true
+    mockserver.enableCORSForAllResponses=true
+    mockserver.initializationJsonPath=/config/initializerJson.json
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: mockserver
+  name: mockserver
+  namespace: mockserver
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mockserver
+  template:
+    metadata:
+      labels:
+        app: mockserver
+      name: mockserver
+    spec:
+      containers:
+      - env:
+        - name: LOG_LEVEL
+          value: INFO
+        - name: SERVER_PORT
+          value: "1080"
+        image: mockserver/mockserver:mockserver-5.11.2
```

Comment:
Please, use the migrated image from our registry

Reply:
Of course. Can you check if locator is fine now?